src/models/GOKU.jl

```diff
@@ -5,8 +5,21 @@
 
 struct GOKU <: LatentDE end
 
+@doc raw"""
+    apply_feature_extractor(encoder, x)
```

Comment:
add the type of encoder   ::Encoder{GOKU}

Reply:
But the encoder can also have type ::Encoder{LatentODE}.  Should I have two separate documentations for ::Encoder{GOKU} and ::Encoder{LatentODE} when everything else in the documentation (i.e. description, arguments) is the same?

Reply:
No, we shouldn't repeat all the description (see [this](https://docs.julialang.org/en/v1/manual/documentation/#Functions-and-Methods)). You are right, we shouldn't include the encoder type here.

Reply:
Actually, I think is better if we add for all the documented functions its type signatures like here: https://github.com/chengchingwen/Transformers.jl/blob/master/src/basic/transformer.jl
(see the Transformer function). So we will repeat only the function name with the different type signatures and then provide only a single general description. Also, you'll have to include this only for one of the methods. So if, for example, you include the documentation in the GOKU methods, you don't have to include it in the LatentODE methods of the same functions. I'm thinking that maybe it is better to move all the documentation of these functions to the LatentODE file, so that the code in GOKU.jl is cleaner at sight. Also, see that we can just describe the arguments in a paragraph style.